aspnet/migration/rc1-to-rc2.rst

```diff
@@ -115,41 +142,31 @@ To:
   public Startup(IHostingEnvironment hostingEnvironment)
   {
      var builder = new ConfigurationBuilder()
-      .SetBasePath(hostingEnvironment.ContentRootPath)
+      .SetBasePath(hostingEnvironment.ContentRootPath);
   }
 
-  
-Namespace and package ID changes
----------------------------------- 
-
-All Microsoft.AspNet.\* namespaces are renamed to Microsoft.AspNetCore.\*. 
-The EntityFramework.\* packages and namespaces are changing to Microsoft.EntityFrameworkCore.\*.
-The ASP.NET Identity packages for Entity Framework Core are changing from Microsoft.AspNet.Identity.EntityFramework.\* to Microsoft.AspNetCore.Identity.EntityFrameworkCore.\*.
-All ASP.NET Core package versions are now 1.0.0-\*.
-Microsoft.Data.Entity.* is now Microsoft.EntityFrameworkCore.*
 
 Working with IIS
 ----------------
 
-HttpPlatformModule
-^^^^^^^^^^^^^^^^^^
-
 ``Microsoft.AspNetCore.IISPlatformHandler`` is now ``Microsoft.AspNetCore.Server.IISIntegration``.
 
-HttpPlatformModule was replaced by ASP.NET Core Module. The web.config created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
+HTTP Platform Handler was replaced by ASP.NET Core Module. The ``web.config`` created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
```

Comment:
I think we said we should use "HttpPlatformHandler" instead of "HTTP Platform Handler".